scipy/stats/tests/test_morestats.py

```diff
@@ -1564,7 +1564,7 @@ def test_gh_6873(self):
         # to only about five significant digits.
         assert_allclose(lam, -0.051654, rtol=1e-5)
 
-    @pytest.mark.parametrize("bounds", [(-1, 1), (0, 1), (-2, -1)])
+    @pytest.mark.parametrize("bounds", [(-1, 1), (1.1, 2), (-2, -1.1)])
```

Comment:
If the bounds don't intersect, then it's more obvious that all of the tests do something different.